frontend/src/css/modules/challenge.scss

```diff
@@ -175,6 +175,14 @@ md-select.md-default-theme .md-select-value span:first-child:after, md-select .m
     color: white;
 }
 
+.md-checkbox-enabled.md-default-theme[selected] .md-icon, .md-checkbox-enabled[selected] .md-icon {
+    background-color: #ffaf4b !important;
```

Comment:
Is it really necessarily important use `!important` here? it is not actually a good practice.

Reply:
Thanks for the review!
I have removed `!important`.